api/src/main/java/gov/va/vro/api/model/mas/ClaimDetailConditions.java

```diff
@@ -0,0 +1,24 @@
+package gov.va.vro.api.model.mas;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Setter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ClaimDetailConditions {
+
+  private String name;
+
+  @NotBlank(message = "Diagnostic Code is required")
+  private String diagnosticcode;
+
+  private String disabilityactiontype;
```

Comment:
Why do we have these all lower case? Isn't Java convention lower camel case? If MAS specified as such I rather use @JsonNaming(PropertyNamingStrategies.LowerCaseStrategy).

Reply:
Setting the naming strategy on the file does not work. I have to specify the property for each field